source/javascripts/controllers/_MainController.js.erb

```diff
@@ -96,6 +96,8 @@ import datadogRumCustomTiming from "../utils/datadogCustomRumTiming.ts";
 			viewModel.menuIds = viewModel.menus.map(function(menu) {
```

Comment:
I think the `viewModel.menuIds` were only used by the tour so we can delete them if different ids will be used

Reply:
These have been taken out. For some reason, when I tried doing it before, it resulted in unexpected behavior. However, it may have been due to other items during development, as it's now working perfectly without the old `menuId`s